samples/cellular/nrf_cloud_multi_service/ysbuild_nrf700x-wifi-conn.conf

```diff
@@ -0,0 +1,7 @@
+SB_CONFIG_WIFI_NRF700X=y
```

Comment:
Why is the file called "ysbuild..."?

Reply:
I honestly have no idea how I managed to swap those two letters!

Reply:
You deleted a letter, but regardless, ok.